classes/class-ledyer-api.php

```diff
@@ -38,6 +39,18 @@ public function create_order_session( $data ) {
 		return ( new Create_Order( compact( 'data' ) ) )->request();
 	}
 
+	/**
+	 * Creates a Ledyer HPP URL.
+	 *
+	 * @param string $session_id The Ledyer Checkout session to use for the HPP request.
+	 * @param int    $order_id WooCommerce order ID.
+	 * @return string
+	 */
+	public function create_ledyer_hpp_url( $session_id ) {
+		$hpp = new Create_HPP();
+		return $hpp->create_hpp_url( $session_id );
+	}
```

Comment:
This isn't an actual API call right? So this should probably be in a separate HPP class, and the `Create_HPP` code should probably be in there too since its not actually a request. Just to avoid future confusion regarding this.